lib/teiserver/protocols/tachyon_v1/auth_out.ex

```diff
@@ -42,10 +42,11 @@ defmodule Teiserver.Protocols.Tachyon.V1.AuthOut do
   end
 
   def do_reply(:verify, {:success, user}) do
+    icons = %{ icons: %{"play_time_rank" => user.rank}}
```

Comment:
Instead of manually adding the icon here, it should be added to the list of icons generated for a user. The rank icon won't be specific to just Tachyon and anything else (e.g. website) that uses the icon list will have to duplicate this code.

Reply:
Fixed.